packages/dx-react-grid/src/plugins/table-column-visibility.jsx

```diff
@@ -18,6 +24,9 @@ const pluginDependencies = [
 const visibleTableColumnsComputed = ({ tableColumns, hiddenColumnNames }) =>
   visibleTableColumns(tableColumns, hiddenColumnNames);
 
+const getColumnTogglingEnabled = (columnExtensions, defaultValue) =>
```

Comment:
Maybe we should rename this Getter to `isColumnTogglingEnabled` because this function who returns a `boolean` value.
Or `columnTogglingEnabled` because other similar Getters don't have `get`

Reply:
This function that returns function, not boolean)

Reply:
Why it returns a function?

Reply:
Right. The Getter's name is "columnTogglingEnabled". There is not the `get` prefix:

```js
<Getter
  name="columnTogglingEnabled"
   value={getColumnTogglingEnabled(columnExtensions, columnTogglingEnabled)}
/>
```

`const getColumnTogglingEnabled = () => {}` calculates Getter's value. Getter's value is a function that takes column name and returns boolean.